### DISCUSSION TITLE
Different wallpaper for different monitor

### DISCUSSION THREAD

**Discussion by syaqil520** [prasanthrangan/hyprdots#1665] [Q&A]
Is there a way to make the monitor to display different wallpaper ? 

as i research sww does not have this capabilities, i try to use hyprpaper but it not working either.

**Comment by kRHYME7** [comment]
swww img -o <outputs> <path/to/img>

**Reply by crabreader** [reply]
Yes this works at first but it is overridden the next time swwwallpaper.sh is executed. It looks like only one wallpaper can be selected at a time?

**Reply by kRHYME7** [reply]
Yes, because of theming, we tried to use the current wallpaper as the template.  You can modify the swwwallpaper script to your own use case.

**Reply by crabreader** [reply]
Ok, I'll see what I can manage.
